pandas/core/frame.py

```diff
@@ -2375,34 +2375,41 @@ def query(self, expr, inplace=False, **kwargs):
             return new_data
 
     def eval(self, expr, inplace=False, **kwargs):
-        """Evaluate an expression in the context of the calling DataFrame
-        instance.
+        """
+        Evaluate expression in the context of the calling DataFrame instance.
+
+        Evaluates a string describing operations on dataframe columns.  This
+        allows `eval` to run arbitrary code, but remember to sanitize your
+        inputs.
```

Comment:
Might be useful to state clearly that this will not evaluate expression within elements of the DataFrame, that it will only evaluate columns.